Sources/MapboxMaps/Style/StyleSourceManager.swift

```diff
@@ -46,11 +47,13 @@ internal final class StyleSourceManager: StyleSourceManagerProtocol {
     internal init(
         styleManager: StyleManagerProtocol,
         mainQueue: DispatchQueueProtocol = DispatchQueue.main,
-        backgroundQueue: DispatchQueueProtocol = DispatchQueue(label: "GeoJSON parsing queue", qos: .userInitiated)
+        backgroundQueue: DispatchQueueProtocol = DispatchQueue(label: "GeoJSON parsing queue", qos: .userInitiated),
+        commonSettings: SettingsServiceInterface = SettingsServiceFactory.getInstance(storageType: .nonPersistent)
```

Comment:
It can be argued that either storage type fits better here. I'd say it's down to the user preference which one to choose. Too bad that common settings don't allow this.